src/main/kotlin/turku/forge/babyproject/config/WebSocketConfig.kt

```diff
@@ -1,25 +1,37 @@
 package turku.forge.babyproject.config
 
+import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.context.annotation.Configuration
 import org.springframework.messaging.simp.config.MessageBrokerRegistry
 import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker
 import org.springframework.web.socket.config.annotation.StompEndpointRegistry
 import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer
+import turku.forge.babyproject.ConfigProperties
+import turku.forge.babyproject.api.API_PATH
+import turku.forge.babyproject.api.CHANNEL_PATH
 
-const val STOMP_ENDPOINT = "/connect"
-const val APPLICATION_DESTINATIONS = "/message"
+
+const val STOMP_ENDPOINT = "$API_PATH/connect"
 
 /**
- * This is a configuration where we extend and override the base `WebSocketMessageBrokerConfigurer`
+ * This class enables the Websockets broker witch
```

Comment:
Don't go witchhunting in the 21st century. :grin: 

```suggestion
 * This class enables the Websockets broker which
```

Reply:
True, It sucks that witch is a gramaticly correct word so not even my IDE that checks for spelling and gramatical errors notices when I got witch hunting 🤣